internal/email/email.go

```diff
@@ -46,12 +63,42 @@ func NewMessage(
 		return nil, fmt.Errorf("error generating UUID: %w", err)
 	}
 
+	// Populate the template data source
+	data := bodyData{
+		AdminName:        sponsorDisplay,
+		OrganizationName: projectDisplay,
+		InvitationURL:    inviteURL,
+		RecipientEmail:   inviteeEmail,
+		MinderURL:        minderURLBase,
+		TermsURL:         DefaultMinderTermsURL,
+		PrivacyURL:       DefaultMinderPrivacyURL,
+		SignInURL:        minderURLBase,
+		RoleName:         role,
+		RoleVerb:         authz.AllRolesVerbs[authz.Role(role)],
+	}
+
+	// Validate the data source template for HTML injection attacks or empty fields
+	err = validateDataSourceTemplate(&data)
```

Comment:
You could add this method to `bodyData` now that the struct is declared at the top-level.  (Probably as `Validate`)